File: app/src/main/java/com/eljaviluki/grindrplus/Logger.java
package com.eljaviluki.grindrplus;


import android.util.Log;

import de.robv.android.xposed.XposedBridge;

public class Logger {
    public static final String TAG = "GrindrPlus";

    public static void xLog(String msg){
        XposedBridge.log(TAG + ": " + msg);
    }

    public static void log(String msg){
        Log.i(TAG, msg);
    }
}

File: app/src/main/java/com/eljaviluki/grindrplus/Logger.kt
package com.eljaviluki.grindrplus

import android.util.Log
import de.robv.android.xposed.XposedBridge

object Logger {
    const val TAG = "GrindrPlus"
    fun xLog(msg: String) {
        XposedBridge.log("$TAG: $msg")
    }

    fun log(msg: String) {
        Log.i(TAG, msg)
    }
}